src/trivia/config.h.cmake

```diff
@@ -299,6 +299,7 @@
 #cmakedefine BUILD_STATIC 1
 #cmakedefine EMBED_LUAZLIB 1
 #cmakedefine EMBED_LUAZIP 1
+#cmakedefine TEST_BUILD 1
```

Comment:
Nits:

1. Shouldn't we update the description of the option in `CMakeLists.txt:821`? It only mentions a memory leak now.
2. Shouldn't we describe what behavior is affected by the option in the documentation request?

Reply:
1. Fixed. The description was mistaken from the beginning.

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1b357f38c..bd9082973 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -818,7 +818,7 @@ elseif(IS_DIRECTORY .git AND GIT)
     )
 endif()

-option(TEST_BUILD "Abort if memory leak is found." OFF)
+option(TEST_BUILD "Use defaults suited for tests" OFF)
 set(ABORT_ON_LEAK_DEFAULT ${TEST_BUILD})
 option(ABORT_ON_LEAK "Abort if memory leak is found." ${ABORT_ON_LEAK_DEFAULT})
```

2. Looks like we don't document build options.